Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -11,7 +11,7 @@ ID_SET=$CIRCLE_ARTIFACTS/id_set.json
 EXTRACT_FOLDER=$(mktemp -d)
 
 if [[ -z "$GCS_MARKET_KEY" ]]; then
```

Comment:
lets skip the whole step in forked PRs so we wont even run this script

@yuvalbenshalom FYI